gyronimo/metrics/morphism_cylindrical.hh

```diff
@@ -0,0 +1,50 @@
+// ::gyronimo:: - gyromotion for the people, by the people -
+// An object-oriented library for gyromotion applications in plasma physics.
+// Copyright (C) 2022 Paulo Rodrigues.
+
+// ::gyronimo:: is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// ::gyronimo:: is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with ::gyronimo::.  If not, see <https://www.gnu.org/licenses/>.
+
+// @morphism_cylindrical.hh, this file is part of ::gyronimo::
+
+#ifndef GYRONIMO_MORPHISM_CYLINDRICAL
+#define GYRONIMO_MORPHISM_CYLINDRICAL
+
+#include <cmath>
```

Comment:
There are no cmath function calls in this file, the include directive should be moved to morphism_cylindrical.cc.

Reply:
I will make the change.